Std/Data/HashMap/Lemmas.lean

```diff
@@ -4,12 +4,21 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Scott Morrison
 -/
 import Std.Data.HashMap.Basic
+import Std.Data.Array.Lemmas
 import Std.Util.ProofWanted
 
 namespace Std.HashMap
 
-@[simp] proof_wanted empty_find? [BEq α] [Hashable α] {a : α} :
-    (∅ : HashMap α β).find? a = none
+namespace Imp
+
+@[simp] theorem empty_buckets : (empty : Imp α β).buckets = ⟨mkArray 8 AssocList.nil, by simp⟩ := rfl
+
+end Imp
+
+@[simp] theorem empty_imp [BEq α] [Hashable α] : (∅ : HashMap α β).1 = Imp.empty := rfl
```

Comment:
It looks like `empty_imp` should be put in the `Imp` namespace similar to `empty_buckets`.

Reply:
Done, does the new name work?